packages/base/Tabs/src/Tab/Tab.tsx

```diff
@@ -45,27 +43,81 @@ export interface Props
   onClick?: TabProps['onClick']
 }
 
-const useStyles = makeStyles<Theme>(styles, { name: 'PicassoTab' })
+const getOpacityClass = (
+  selected: boolean,
+  disabled: boolean,
+  orientation: 'horizontal' | 'vertical'
+) => {
+  if (disabled) {
+    return 'opacity-50'
+  }
+
+  if (selected || orientation === 'vertical') {
+    return 'opacity-100 '
+  }
 
-export const Tab = forwardRef<HTMLDivElement, Props>(function Tab(props, ref) {
+  return 'opacity-70'
+}
+
+const wrapperClassesByOrientation = {
+  horizontal: 'inline-flex items-center flex-col justify-center',
+  vertical: 'block',
+}
+
+const rootClassesByOrientation = (selected: boolean) => ({
+  horizontal: [
+    'm-0 [&:not(:last-child)]:mr-8 pt-[0.5625rem] pb-[0.4375rem] px-0',
+    'text-center bg-transparent transition-shadow z-10 rounded-none',
+    'text-black',
+    selected && 'shadow-blue-500 shadow-[inset_0_-2px_0]',
+  ],
+  vertical: [
+    ' first:mt-4 last:mb-4 my-1 mx-0 py-2 px-4',
+    'text-left rounded-l-sm rounded-r-none transition-all',
+    'w-full overflow-hidden',
+    selected && 'shadow-1',
+    selected && [
+      'before:absolute',
+      'before:content-[""]',
+      'before:bottom-0',
+      'before:left-0',
+      'before:top-0',
+      'before:w-[3px]',
+      'before:bg-blue-500',
+    ],
```

Comment:
Does passing an array work with `twMerge`? Didn't know that
Why not use a single string here?
```suggestion
    selected &&
      'before:absolute before:content-[""] before:bottom-0 before:left-0 before:top-0 before:w-[3px] before:bg-blue-500',
```

I guess it boils down to a stylistic choice, but we didn't use arrays anywhere else, AFAIK. Using strings might be more consistent. I don't have a strong opinion here, if you prefer arrays it is fine to me

Reply:
We are using arrays in some places and yeah, twMerge can work even with multiple depths of array, it will flatten it. To me, as all these lines have the same prefix, it is better readable